File: javascript/gemini/p144.js
async function handleUnsubscribe(email, db) {
  // 1. Check database
  const user = await db.query('SELECT id FROM users WHERE email = ?', [email]);
  
  if (user.length === 0) {
    return 0;
  }

  // 2. Call PHP script
  try {
    const response = await fetch(`https://api.example.com/unsubscribe.php?email=${email}`);
    const result = await response.text();

    // 3. Logic: If result is "0", return 1
    return result.trim() === "0" ? 1 : 0;
  } catch (error) {
    return 0;
  }
}

File: python/gemini/p144.py
import requests

def manage_subscription(email, db_connection):
    # 1. Check if email exists in DB
    cursor = db_connection.cursor()
    cursor.execute("SELECT id FROM users WHERE email = %s", (email,))
    
    if not cursor.fetchone():
        return 0
    
    # 2. Call the PHP script
    try:
        response = requests.get(f"https://api.example.com/unsubscribe.php?email={email}")
        php_result = response.text.strip()
        
        # 3. Logic: If PHP returns 0, return 1. Else return 0.
        return 1 if php_result == "0" else 0
    except:
        return 0